docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+In order to update a DID Document, either an Integration or a Differentation DID message needs to be generated. It is RECOMMENDED to use only Integration DID messages if the DID Document is updated very infrequently and it is expected to never go beyond 100 updates in the lifetime of the DID. If that is not the case, it is RECOMMENDED to use as much Differentiation DID messages instead with a maximum of around 100 updates per Diff chain. 
+
+#### Creating an Integration DID message
+
+An Integration DID message is unrestricted in its operations and may add or remove any fields to the DID Document. In order to query the DID Document, every Integration DID message must be processed, therefore it is RECOMMENDED to reduce the usage of these messages unless the DID Document in updated very infrequently. 
+
+In order to create a valid Integeration DID message, the following steps must be executed:
+* Create a new DID Document that contains all the new target state. This SHOULD include the new desired changes and all the changes inside the previous Diff Chain, otherwise these changes are lost!
+* Retrieve the IOTA MessageId from the previous Integration DID message and a keypair for signing the DID Document.
+* Set the `previous_message_id` field to the IOTA MessageId value.
+* Create a `proof` object referencing the public key used inside the `verificationMethod` field and the signature suite in the `type` field. 
+* Create a cryptographic signature using the same keypair and add the result in the appropriate field(s) inside the `proof` object.
+
+#### Creating an Differentiation DID message
+
+An Differentiation DID message is restricted in its usage. It may not update any signing keys that are used in the Diff chain. If this is desireable, it is REQUIRED to use an Integretation DID message. If the current Diff chain becomes to long (currently RECOMMENDED to end at a length of 100), it is RECOMMENDED to use a single Integration DID message to reset this length.
+
+In order to create a valid Integration DID message, the following steps must be executed:
+* Create and serialize a `diff` JSON object that contains all the changes. 
+* Set the `did` field to the DID value that this update applies to.
+* Retrieve the IOTA MessageId from the previous Diff chain message, or Integration DID message if this message is the first in the Diff chain. and a keypair for signing the DID Document.
+* Set the `previous_message_id` field to the IOTA MessageId value.
+* Create a `proof` object referencing the public key used inside the `verificationMethod` field and the signature suite in the `type` field. 
+* Create a cryptographic signature using the same keypair and add the result in the appropriate field(s) inside the `proof` object.
+
+### Delete
+
+In order to deactivate a DID document, an Integration DID message must be published that removes all content from a DID Document, effectively deactivating the DID Document. Keep in mind that this is irreversable. 
```

Comment:
Shouldn't that be signed?

Reply:
Doesn't say it isn't? It removes the content. But I'll clarify.